README.md

```diff
@@ -1,2 +1,91 @@
-# curveadm
+# CurveAdm
 Deploy and manage CurveFS cluster
```

Comment:
Deploy and manage Curve cluster (only support CurveFS now, CurveBS will be supported later).

Reply:
> Deploy and manage Curve cluster (only support CurveFS now, CurveBS will be supported later).

fixed.